### PR TITLE
1- define release process

### PR DESCRIPTION
**Pull request by Emily-Jiang**
Signed-off-by: Emily Jiang <emijiang6@googlemail.com>

### CODE REVIEW

File: .github/ISSUE_TEMPLATE/specreview_checklist.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/specreview_checklist.md:2]: What about filling in some of the preamble for filling in some of the fields of the Issue, like what Scott had... Maybe not labels?  But, we need something to make this easier to understand for the people creating these issues.

```
name: Specification Review Issue
about: Start a request for a specification creation/plan/release... lifecycle review
title: [MP Component] [ Version] Specification Review
labels: creation,plan,progress,release
assignees: ''
```

File: .github/ISSUE_TEMPLATE/specreview_checklist.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/specreview_checklist.md:3]: We shouldn't be defining a PR here...  We need to define the Issue template.  I'll talk with you offline so that we're on the same page...

File: .github/ISSUE_TEMPLATE/compatible-certification-request.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/compatible-certification-request.md:1]: The Compatible Certification Request should be filed against the Specification repository, not the microprofile-wg repository.  So, it doesn't really belong here...  Each of the Spec repositories should have this template available for creating CCRs.  We can use this one as a "template for a template", but we wouldn't actually use it here.

Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/compatible-certification-request.md:1]: You should include the metadata header to make this template more useful, just like the spec_issue.md template:
```
---
name: Compatible Certification Request
about: Start a request for a compatible certification 
title: ''
label: Certification
assignees: ''
---
```

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:4]: title: \[MP Component] \[Version] Specification Review

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:5]: label: Creation Review, Plan Review, Progress Review, Release Review

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:3]: about: Start a request for a specification creation/plan/progress/release (lifecycle) review

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:10]: Missing some words...  Maybe something like this:  "..create issues in the [MicroProfile-WG repository](https://github.com/microprofile/microprofile-wg) with.."

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:12]: Need the other valid labels:  `Creation Review`, `Plan Review`, `Progress Review`, `Release Review`

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:20]: Maybe add the following as a comment so that people know what is expected:
`<!-- (eg, https://oss.sonatype.org/content/repositories/orgeclipsemicroprofile-1357/org/eclipse/microprofile/config/) -->`

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:21]: get rid of space between "] ("

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:22]: get rid of space between "] ("

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:30]: "project", not "projece"

Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:30]: Should you also italicize `_version_` ?

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:31]: FYI, when we do put the promotion scripts in place, the "apidocs" directory can not be promoted.  The scripts only work with files (zips, jars, etc).  Maybe something can be changed down the road, but just letting you know the current limitation.

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:39]: A member of the Steering/Spec Committee should be initiating the ballot, not the producer of the Spec.  So, once we get an informal sign-off via this Issue (maybe via thumbs-up), then a ballot can be initiated.  We don't want the producer of the Spec to decide when it's ready for ballot.  We may end up with several invalid ballots that way...

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:41]: Highlight that the CCR should be filed against the repository of the Specification.

Remove the space between "] (".

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:42]: This can now be removed because it's covered by the CCR.

**Review by kwsutter — CHANGES_REQUESTED** [review]
A few things to address before we merge...  Thanks, @Emily-Jiang!

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (Emily-Jiang) [reply .github/ISSUE_TEMPLATE/spec_issue.md:3]: I am not sure about this as I don't think we really need for different stages.

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (Emily-Jiang) [reply .github/ISSUE_TEMPLATE/spec_issue.md:42]: In this case, I can move the CCR to microprofile-config repo. The reason I put here so that I can reference that file.

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (Emily-Jiang) [reply .github/ISSUE_TEMPLATE/spec_issue.md:31]: ok, thanks. That will be shame as the javadoc will not be browsable anymore. Maybe we can point to the staging. Anyway, we can figure it out later.

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (Emily-Jiang) [reply .github/ISSUE_TEMPLATE/spec_issue.md:39]: good point. udpated

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (Emily-Jiang) [reply .github/ISSUE_TEMPLATE/spec_issue.md:5]: We settle on Release Review for now. If we need other reviews, we can then update the template.

**Review by radcortez — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [reply .github/ISSUE_TEMPLATE/spec_issue.md]: Actually, the template should probably be documented somewhere on our MicroProfile pages...  Not sure where, but we should probably start an "operations guide" explaining these types of steps.

Moving it to the Config repo is fine for now.  We can cut-and-paste to other repos.

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:35]: git rid of the "[x]"...

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (kwsutter) [inline .github/ISSUE_TEMPLATE/spec_issue.md:39]: "initiates" (not initiating)

Also, related to my other comment about an operations guide... For Jakarta EE, we have sample text that should be used for these ballots.  We should do something similar for MicroProfile.  For an example, here's the email that we sent out for the Jakarta EE 9 Platform ballot:
https://www.eclipse.org/lists/jakarta.ee-spec/msg01300.html  (now, of course, these are PRs instead of Issues, but you get the general idea)

**Review by kwsutter — APPROVED** [review]
Just a couple of (hopefully) minor items...  I will still Approve so that we can make progress, but we should clean these up before merging.  Thanks!

File: .github/ISSUE_TEMPLATE/spec_issue.md
Comment (Emily-Jiang) [reply .github/ISSUE_TEMPLATE/spec_issue.md:39]: Thanks Kevin! When we send the first accepted email, we will then add a link to that example in this template for others to follow.

**Pull request merged**
